go-controller/pkg/node/node.go

```diff
@@ -297,11 +327,16 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 					klog.Fatalf("Error while running upgrade controller: %v", err)
 				}
 				// upgrade complete now see what needs upgrading
+				// migrate service route from ovn-k8s-mp0 to shared gw bridge
 				if initialTopoVersion < types.OvnHostToSvcOFTopoVersion {
 					if err := upgradeServiceRoute(bridgeName); err != nil {
 						klog.Fatalf("Failed to upgrade service route for node, error: %v", err)
 					}
 				}
+				// ensure CNI support for port binding built into OVN, as masters have been upgraded
+				if initialTopoVersion < types.OvnPortBindingTopoVersion {
```

Comment:
this is inside of an "if shared gw" block due to the first thing I added in this upgrade mechanism was around service upgrade for DGP -> OF. I'll change this so this applies to both local and shared gw mode